homeassistant/components/sensor/wunderground.py

```diff
@@ -764,28 +770,31 @@ def request_feature(self, feature):
 
     def _build_url(self, baseurl=_RESOURCE):
         url = baseurl.format(
-            self._api_key, "/".join(self._features), self._lang)
+            self._api_key, '/'.join(sorted(self._features)), self._lang)
```

Comment:
`self._features` needs to be sorted in order to make the generated URL stable for tests - so that it doesn't depend on the order of execution.